Smith_Ben/Assignments/Dojo_Survey copy/templates/result.html

```diff
@@ -0,0 +1,24 @@
+<html>
+    <head>
+       <title>Dojo Suvey</title>
+    </head>
+    <body>
+			{% with messages = get_flashed_messages() %}
+			{% if messages %}
+			{% for message in messages %}
+			<p>{{message}}</p>
+			{% endfor %}
+			{% endif %}
+			{% endwith %}
+
+		<h1><u>Submitted Info</u></h1>
+		<p>Name: {{ session['name'] }}</p>
+		<p>Dojo Location: {{ session['Location'] }}</p>
+		<p>Favorite Language: {{ session['Language'] }}</p>
+		<p>Comment: {{ session['comment'] }}</p>
+		<form action='/back'>        
```

Comment:
Since you're only going back to the index page all you need is a button or a tag with href='/'.  That way it just goes directly to your root route on the server.py file and you won't need the '/back route